docs/issue_template.md

```diff
@@ -0,0 +1,22 @@
+# Checklist for author
+
+Remove items that aren't applicable.
+
+## Issue Type
+
+- [ ] Bug Report
+- [ ] Feature Request (requested by stakeholders)
+- [ ] Feature Proposal (possible improvements)
+
+## Issue Description
+
+### Input here issue description
+
+### Extra information
+
+- [ ] Login role (to reproduce, what role should I log in with)
+- [ ] Possible screenshots
+- [ ] For UI-related issues, you might include references to design or related UX/UI study
+- [ ] For the user feedback feature, you might include the name of the user (organisation) requesting a feature
+- [ ] Importance - you might mark this if issue is critical / needs to be fixed by nextv release
```

Comment:
We've not been tracking importance with labels or anything like that yet, so I think we can drop this line for now.

Reply:
I'm not completely sure the checkboxes actually do something here at all, except for the bug or feature request.

Reply:
From my perspective, they are more on the "checklist/reminder" side, rather then accomplishing any particular goal. like, if you are reporting a bug, it reminds you to mention what credentials to log in with, for example